src/modules/Lan.rb

```diff
@@ -800,7 +800,8 @@ def autoinst_settings?(section)
       !section.start_immediately.nil? ||
         !section.keep_install_network.nil? ||
         !section.setup_before_proposal.nil? ||
-        !section.virt_bridge_proposal.nil?
+        !section.virt_bridge_proposal.nil? ||
+        !section.strict_ip_check_timeout.nil?
```

Comment:
np: Perhaps it is time to start thinking about applying the De Morgan's law. :-)